src/Services/HardwareService.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Taecontrol\Larvis\Services;
+
+use Taecontrol\Larvis\Exceptions\CpuHealthException;
+use Taecontrol\Larvis\Exceptions\DiskHealthException;
+use Taecontrol\Larvis\ValueObjects\Data\HardwareData;
+use Taecontrol\Larvis\Exceptions\MemoryHealthException;
+
+class HardwareService
+{
+    public function getHardwareData()
+    {
+        $cpuLoad = $this->getCpuLoadUsage();
+        $memory = $this->getMemoryUsage();
+        $disk = $this->getDiskUsage();
+
+        $HardwareData = new HardwareData($cpuLoad, $memory, $disk);
+
+        return $HardwareData->toArray();
```

Comment:
If this is the only place where you use the value object it can be removed from the source code.